.github/workflows/sonarqube.yml

```
@@ -0,0 +1,32 @@
+name: SonarQube
+on:
+  workflow_run:
+    workflows: [ "Tests" ]
+    types:
+      - completed
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.workflow_run.head_branch }}
+  cancel-in-progress: true
+jobs:
+  sonarqube:
+    name: SonarQube
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: ${{ github.event.workflow_run.head_repository.full_name }}
+          ref: ${{ github.event.workflow_run.head_branch }} # checkout commit that triggered this workflow
+          fetch-depth: 0 # Shallow clones should be disabled for a better relevancy of analysis
+      - uses: matrix-org/setup-python-poetry@v1
+      - name: SonarCloud Scan
+        uses: matrix-org/sonarcloud-workflow-action@v2.1
```

```suggestion
        uses: matrix-org/sonarcloud-workflow-action@v2.2
```

2.1 had an issue where the sonar.projectVersion wasn't being passed correctly